test.sh

```diff
@@ -1,80 +1,48 @@
-#!/usr/bin/env bash
-
-# This file contains logic for integration tests which are executed by CI upon pull requests to
-# dcos-commons. The script builds each framework, packages and uploads it, then runs its
-# integration tests against a newly-launched cluster.
-
-# Exit immediately on errors -- the helper scripts all emit github statuses internally
-set -e
-
-function run_framework_tests {
-    framework=$1
-    FRAMEWORK_DIR=${REPO_ROOT_DIR}/frameworks/${framework}
-
-    # Build/upload framework scheduler artifact if one is not directly provided:
-    if [ -z "${!STUB_UNIVERSE_URL}" ]; then
-        STUB_UNIVERSE_URL=$(echo "${framework}_STUB_UNIVERSE_URL" | awk '{print toupper($0)}')
-        # Build/upload framework scheduler:
-        UNIVERSE_URL_PATH=$FRAMEWORK_DIR/${framework}-universe-url
-        UNIVERSE_URL_PATH=$UNIVERSE_URL_PATH ${FRAMEWORK_DIR}/build.sh aws
-
-        if [ ! -f "$UNIVERSE_URL_PATH" ]; then
-            echo "Missing universe URL file: $UNIVERSE_URL_PATH"
-            exit 1
-        fi
-        export STUB_UNIVERSE_URL=$(cat $UNIVERSE_URL_PATH)
-        rm -f $UNIVERSE_URL_PATH
-        echo "Built/uploaded stub universe: $STUB_UNIVERSE_URL"
-    else
-        echo "Using provided STUB_UNIVERSE_URL: $STUB_UNIVERSE_URL"
-    fi
-
-    echo Security: $SECURITY
-    if [ "$SECURITY" = "strict" ]; then
-        ${REPO_ROOT_DIR}/tools/setup_permissions.sh root ${framework}-role
-        # include foldered roles (tests exercise with /test/integration/svcname):
-        ${REPO_ROOT_DIR}/tools/setup_permissions.sh root test__integration__${framework}-role
-    fi
-
-    # Run shakedown tests in framework directory:
-    TEST_GITHUB_LABEL="${framework}" ${REPO_ROOT_DIR}/tools/run_tests.py shakedown ${FRAMEWORK_DIR}/tests/
-}
-
-echo "Beginning integration tests at "`date`
-
-REPO_ROOT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-
-cd $REPO_ROOT_DIR
-
-# Get a CCM cluster if not already configured (see available settings in dcos-commons/tools/README.md):
-if [ -z "$CLUSTER_URL" ]; then
-    echo "CLUSTER_URL is empty/unset, launching new cluster."
-    export CCM_AGENTS=6
-    CLUSTER_INFO=$(${REPO_ROOT_DIR}/tools/launch_ccm_cluster.py)
-    echo "Launched cluster: ${CLUSTER_INFO}"
-    # jq emits json strings by default: "value".  Use --raw-output to get value without quotes
-    export CLUSTER_URL=$(echo "${CLUSTER_INFO}" | jq --raw-output .url)
-    export CLUSTER_ID=$(echo "${CLUSTER_INFO}" | jq .id)
-    export CLUSTER_AUTH_TOKEN=$(echo "${CLUSTER_INFO}" | jq --raw-output .auth_token)
-    CLUSTER_CREATED="true"
-else
-    echo "Using provided CLUSTER_URL as cluster: $CLUSTER_URL"
-    CLUSTER_CREATED=""
+#!/bin/bash
```

Comment:
Sounds like this script is intended to only be used in CI. Maybe give it a CI-specific filename?